controller/api/models.py

```diff
@@ -519,7 +536,6 @@ class Push(UuidAuditedModel):
     class Meta:
         get_latest_by = 'created'
         ordering = ['-created']
-        unique_together = (('app', 'uuid'),)
```

Comment:
Why did we remove the `unique_together` Meta fields? This effectively changes the database schema by dropping a `UNIQUE` constraint: https://docs.djangoproject.com/en/1.6/ref/models/options/#unique-together